scss/_patterns_links.scss

```diff
@@ -37,20 +37,30 @@
   }
 
   .p-top {
-    border-bottom: 1px dotted $color-mid-light;
-    clear: both;
-    margin: 20px 0;
+    align-items: center;
+    display: flex;
+    margin-top: -$spv-outer--scaleable;
```

Comment:
This is risky, if you put a dark strip on top it overlaps it:

![image](https://user-images.githubusercontent.com/2741678/114043438-30629380-987e-11eb-8188-52eb6e6fe3bb.png)